### ISSUE TITLE
[SITE] Add logging?

### ISSUE THREAD

**Issue by RoelN** [Wakamai-Fondue/wakamai-fondue#28]
It would be nice to keep logs on which fonts are dropped, which types, etc. All with as little privacy-sensitive data as possible.

Twitter thread: https://twitter.com/PixelAmbacht/status/1280104631750795264

Immediately a good reason to opt-out:

> I often use Wakamai Fondue for unreleased/upcoming typefaces, so an opt-out would be preferred — but some kind of summary could definitely be interesting to see

**Comment by RoelN** [comment]
Collecting data on the fonts themselves seems undesirable.

But we can collect some data about [usage](https://twitter.com/davelab6/status/1280470193001713664). How to determine usage with non-interactive elements though?

**Comment by davelab6** [comment]
For each thing that's detected - each language, each ot feature, each axis tag, each axis range - you could increment a counter.

Then you can use this aggregate data to know what is more commonly detected and therefore should have more/better treatment. 

Eg, if small caps are very common but slashed zero is rare, a prominent section that shows off in detail what the former looks like is going to be more worthwhile developing than the latter.

**Comment by alerque** [comment]
> a good reason to opt-out

No, as an offline-first app that specifically states up front that no fonts are uploaded, this is a good reason to make any usage reporting **opt-in**.

**Comment by alerque** [comment]
> if small caps are very common but slashed zero is rare, a prominent section that shows off in detail what the former looks like is going to be more worthwhile developing than the latter.

You could probably get a better idea of what features were rare vs. common by surveying recently released fonts. Also you might want to do the opposite: surface unusual features more prominently than run-of-the-mill features.

Neither of those points are to say collecting usage stats is bad ... I think it can be a great thing. I'd focus more on usage patterns on the page than on collecting data about the fonts people submit, and I would absolutely make it opt-in, but sure it would help inform development and UI decisions.
